### PR TITLE
fix(QField): fix label backface visibility after transform (fix #12172)

### LINKED ISSUE #12172: QInput dark mode error message transition
### What happened?

The transition for the error message on `QInput` with enabled dark mode has a weird transition bug where it seems to not keep the color or loosing anti-aliasing in the transitory state. I'm not sure if this is happening without dark mode too as it might be hard to see there.

This issue is showing up in Chrome for me. In Firefox the same is happening but not as extreme as in Chrome and only for the placeholder, not for the error message itself.

transition-bug

### What did you expect to happen?

It should be a smooth transition without changing colors / loosing font smoothing

### Reproduction URL

https://codepen.io/squareloop1/pen/gOXYKpd

### How to reproduce?

1. Go to reproduction link
2. Trigger the validation by focussing and bluring the input with mouse clicks inside and outside of the input

### Flavour

Quasar CLI (@quasar/cli | @quasar/app)

### Areas

Components (quasar)

### Platforms/Browsers

Chrome

### Quasar info output

_No response_

### Relevant log output

_No response_

### Additional context

Chrome 97.0.4692.71 64bit (Windows)
Maybe related to #11975

### PR DESCRIPTION
**Pull request by gregadro**
**What kind of change does this PR introduce?** 

- [x] Bugfix
- [ ] Feature
- [ ] Documentation
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** 

- [ ] Yes
- [x] No

**The PR fulfills these requirements:**

- [x] It's submitted to the `dev` branch (or `v[X]` branch)
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix: #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] It's been tested on a Cordova (iOS, Android) app
- [ ] It's been tested on a Electron app
- [ ] Any necessary documentation has been added or updated [in the docs](https://github.com/quasarframework/quasar/tree/dev/docs)  or explained in the PR's description.

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to [start a new feature discussion](https://github.com/quasarframework/quasar/discussions/new?category=ideas-proposals) first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

**Comment by gregadro** [comment]
Issue: #12172

**Comment by rstoenescu** [comment]
Thanks for contributing!

File: ui/src/components/field/QField.sass [hunk 1]
```diff
@@ -190,11 +191,13 @@ $field-transition-label-right-up: .324s cubic-bezier(.4,0,.2,1)
     text-transform: inherit
     transform-origin: left top
     transition: transform $field-transition, max-width $field-transition-label-right-up
+    backface-visibility: hidden
 
   &--float .q-field__label
     max-width: 133%
     transform: translateY(-40%) scale(.75)
     transition: transform $field-transition, max-width $field-transition-label-right-down
+    backface-visibility: hidden
```
Comment (pdanpdan) [inline ui/src/components/field/QField.sass:200]: Why is this needed, it is already set at line 194?
Am I missing something (like a quirk that makes this required)?

Comment (gregadro) [reply ui/src/components/field/QField.sass:200]: I was sure backface visibility was broken again after transforming on Chrome(when getting `q-field--float` class). Yesterday I checked your code from 8dccbd4 and it worked without line 200. So I guess it is not really needed, sorry. Thank you for your review.

Comment (pdanpdan) [reply ui/src/components/field/QField.sass:200]: No problem, I just wanted to check if it was required or not :)
Thank you
